README.md

```diff
@@ -79,6 +79,22 @@ The debug logger can be configured to be used for more than one library. In exam
 
 ``DEBUG=ibm-cloud-sdk-core:debug,other-lib:debug``
 
+## Cookie Jar Support
+By default, cookies are not supported in the sdk core requests.  If your SDK would benefit from this functionality, simply instantiate a cookie jar and pass it in the object containing configuration options to the `BaseService` class, as show below.
+
+```typescript
+import tough = require('tough-cookie');
+
+class MyClass extends BaseService {
+  constructor(options: MyOptions) {
+    options.jar = new tough.CookieJar();
+    super(options)
+  }
+}
+```
+
+The example above uses [Tough Cookie](https://www.npmjs.com/package/tough-cookie) to provide these capabilities, but other cookie jar libraries can be used. Alternatively providing `jar: true`, will cause Axios to [instantiate a cookie jar](https://github.com/axios/axios#request-config).
```

Comment:
I don't really think we should link the `axios` config after this statement - `jar` is not a documented option. I think it's a little confusing.

Reply:
I am also confused by this...  The cookie wrapper library references that axios does this...  but I do not see anything in the Axios side that can do this. Per  https://github.com/3846masa/axios-cookiejar-support#extended-request-config

Reply:
I interpret that section of their README to mean that wrapping the `axios` with their tool provides an extended request configuration with those new options. I think it's clear enough to say what you're saying without attaching any links to it.